controllers/chat/group/leaveGroupMember.js

```diff
@@ -56,6 +50,12 @@ const leaveGroupMember = async (req, res) => {
       }
     );
 
+    const response = await Getstream.chat.removeGroupMember(channelId, userId);
+    const {channel, channelApiResponse} = response.data || {};
+
+    const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
+      await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling seems to be missing here. If `Getstream.chat.removeGroupMember(channelId, userId)` or `BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true)` fails, it could lead to unhandled exceptions. It would be better to wrap these calls in a try-catch block to handle potential errors gracefully.
```diff
+    try {
       const response = await Getstream.chat.removeGroupMember(channelId, userId);
       const {channel, channelApiResponse} = response.data || {};
 
       const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
         await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
+    } catch (error) {
+      console.error('Error while removing group member or updating channel members:', error);
+      throw error;
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->